mmdet3d/ops/pointnet_modules/point_sa_module.py

```diff
@@ -77,8 +79,10 @@ def __init__(self,
         self.fps_mod_list = fps_mod
         self.fps_sample_range_list = fps_sample_range_list
 
-        self.points_sampler = Points_Sampler(self.num_point, self.fps_mod_list,
-                                             self.fps_sample_range_list)
+        self.points_sampler = Points_Sampler(
```

Comment:
May use a more regular style. The current logic is not reader-friendly. We can simply use
```
if xxx:
    xxx
else:
    xxx

```

Reply:
```
if self.num_point is not None:
    self.points_sampler = Points_Sampler(xxx)
else:
    self.points_sampler = None
```